### ISSUE TITLE
"ESC" to dismiss the backstage

### ISSUE THREAD

**Issue by yutao-huang** [OfficeDev/script-lab#333]
When the backstage menu is open, I should be able to hit "ESC" to dismiss it .

**Comment by ElizabethSamuel-MSFT** [comment]
Closing all issues since this repo is being archived and no longer maintained.
